### PR TITLE
RStudio.RStudio.OpenSource version 2021.09.1-372

### PR DESCRIPTION
**Pull request by dpprdan**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/45344)

### CODE REVIEW

File: manifests/r/RStudio/RStudio/OpenSource/2021.09.1-372/RStudio.RStudio.OpenSource.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: RStudio.RStudio.OpenSource
+PackageVersion: 2021.09.1-372
```
Comment (jedieaston) [inline manifests/r/RStudio/RStudio/OpenSource/2021.09.1-372/RStudio.RStudio.OpenSource.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2021.09.1+372
```

File: manifests/r/RStudio/RStudio/OpenSource/2021.09.1-372/RStudio.RStudio.OpenSource.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: RStudio.RStudio.OpenSource
+PackageVersion: 2021.09.1-372
```
Comment (jedieaston) [inline manifests/r/RStudio/RStudio/OpenSource/2021.09.1-372/RStudio.RStudio.OpenSource.yaml:5]:
```suggestion
PackageVersion: 2021.09.1+372
```

**Review by jedieaston — CHANGES_REQUESTED** [review]
After these quick fixes, you'll need to rename the folder from 2021.09.1-372 to 2021.09.1+372.

Hot tip to change the folder name: Press "." on your keyboard when you have GitHub open to this PR in your browser, and a web version of VSCode will launch.
 
 Once it loads, find the files you added, and rename the folder. Then use the source control button on the left to commit (it's automatically pushed).
 
 image

**Comment by dpprdan** [comment]
@jedieaston Good catch, thanks!

**Review by OfficialEsco — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**